src/qml/qgismobileapp.qml

```diff
@@ -316,7 +316,7 @@ ApplicationWindow {
               }
               else
               {
-                  if( !overlayFeatureFormDrawer.visible )
+                  if ( !overlayFeatureFormDrawer.visible || !featureForm.canvasOperationRequested )
```

Comment:
```suggestion
                  if (!overlayFeatureFormDrawer.visible || !featureForm.canvasOperationRequested)
```
Following the QML style, the rule is "space after keywords, no space in brackets".